app/src/main/res/layout/fragment_map.xml

```diff
@@ -0,0 +1,7 @@
+<fragment xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Is there any particular reason for using Fragments instead of Composables ? I think it would be better to keep it uniform if possible, but it's no big deal.

Reply:
I have just put there in case in the future one would really wants to use google maps plateform, but I can remove them